gluonnlp/model/transformer.py

```diff
@@ -210,138 +253,18 @@ def hybrid_forward(self, F, inputs, mask=None):  # pylint: disable=arguments-dif
         return outputs, additional_outputs
 
 
-class TransformerDecoderCell(HybridBlock):
```

Comment:
I made no changes to Decoders. They are not exposed as public APIs either. I simply moved it around.